draft-irtf-cfrg-opaque.md

```diff
@@ -1009,9 +1017,9 @@ IKM = concat(epkU^eskS, epkU^skS, pkU^eskS)
 Clients and servers use keys Km2 and Km3 in computing KE2.mac and KE3.mac,
 respectively. These values are computed as follows:
 
-- KE2.mac = HMAC(Km2, Hash(concat(preamble, KE2.enc_server_info)), where
+- KE2.mac = MAC(Km2, Extract(concat(preamble, KE2.enc_server_info)), where
   preamble is as defined in {{derive-3dh}}.
-- KE3.mac = HMAC(Km3, Hash(concat(preamble, KE2.enc_server_info, KE2.mac)),
+- KE3.mac = MAC(Km3, Extract(concat(preamble, KE2.enc_server_info, KE2.mac)),
```

Comment:
@chris-wood I missed this. Why are we moving from a running hash to an Extract()?

Reply:
I missed this too -- I meant to revert this to `Hash`. Can you file an issue (and send PR)? :)